test/unit/attribute/CrossChainOwnable_Arbitrum.test.ts

```diff
@@ -91,6 +91,10 @@ describe('CrossChainOwnable_Arbitrum', () => {
       expect(await ownable.owner()).to.equal(xChainOwner.address)
       expect(await ownable.pendingOwner()).to.equal(ethers.constants.AddressZero)
       expect(await ownable.crossChainRestricted()).to.equal(true)
+
+      // Transfer owner again
+      await ownable.connect(user).updatePendingOwner(xChainOwner.address)
```

Comment:
maybe leave a note that during a valid cross-chain call, the underlying msg.sender is ignored, had to go back and verify why this was safe.

Reply:
Added in [4cffdeb](https://github.com/equilibria-xyz/root/pull/106/commits/4cffdeb06c13f3cc2f08a534b11ce739533b1d17)